Protocol/ErrorMessages.xml

```diff
@@ -9643,6 +9643,108 @@
 							</ErrorMessage>
 						</ErrorMessages>
 					</Check>
+					<Check id="75">
+						<Name namespace="Protocol.Params.Param.Interprete.Exceptions.Exception.Display">CheckStateAttribute</Name>
+						<ErrorMessages>
+							<ErrorMessage id="1">
+								<Name>UnrecommendedEnabledValue</Name>
+								<GroupDescription />
+								<Description>
+									<Format>Exception with state "Enabled". Param {0}.</Format>
```

Comment:
Enabled & Disabled should be lower case everywhere as that is the exact value in the xml.
Also, use single quotes instead of double ones.
Add single quotes around {0}